docs/guides/cli/subdomain.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Subdomain/Region
+description: Connecting to your local environment
+icon: compass
+---
+
+When you start the CLI the services are exposed similarly to the way they are exposed in the [cloud](/platform/subdomain). For instance, the following information is shown on your terminal after running `nhost up`
+
+```
+> nhost up
+...
+URLs:
+- Postgres:             postgres://postgres:postgres@localhost:5432/local
+- Hasura:               https://local.hasura.local.nhost.run
+- GraphQL:              https://local.graphql.local.nhost.run
+- Auth:                 https://local.auth.local.nhost.run
+- Storage:              https://local.storage.local.nhost.run
+- Functions:            https://local.functions.local.nhost.run
+- Dashboard:            https://local.dashboard.local.nhost.run
+- Mailhog:              https://local.mailhog.local.nhost.run
+
+SDK Configuration:]
+ Subdomain:             local
+ Region:                local
+```
+
+There you can see the various URLs you can use to access each service plus the region and subdomain you can use to configure the SDK:
+
+```ts
+// Create a new Nhost client for local development.
+const nhost = new NhostClient(
+    { region: 'local', subdomain: 'local' }
+)
+```
+
+The domains in the URLs above will all return the IP address for localhost, `127.0.0.1`, which should suffice for most development environments. For instance:
+
+```
+> host local.auth.local.nhost.run
+local.auth.local.nhost.run has address 127.0.0.1
+```
+
+However, those URLs are powered by a dynamic DNS that can return any IPv4 address you need, you just need to replace the subdomain `local` with a `subdomain` that contains the 4 octets of the IPv4 adress you want separated by `-`. For instance:
+
+```
+> host 192-168-100-1.auth.local.nhost.run
+192-168-100-1.auth.local.nhost.run has address 192.168.100.1
+
+> host 10-10-1-108.auth.local.nhost.run
+10-10-1-108.auth.local.nhost.run has address 10.10.1.108
+```
+
+This is useful if you need to connect to your environment from a different device, a VM or a mobile device emulator.
+
+To make use of this functionality using the SDK you can specify the subdomain that resolves your IP:
+
+
+```ts
+// Create a new Nhost client for local development.
+const nhost = new NhostClient(
+    { region: 'local', subdomain: '192-168-1-108' }
+)
+```
+
+<Warning>
+If you are trying to connect to your local environment from an external device or VM make sure that:
+
+- The IP address you are using is reachable from this device/VM
+- That your firewall isn't blocking requests
+</Warning>
+
+<Warning>
+If you are testing a social provider don't forget you will need to configure the callback URL to match the subdomain/region you are using.
+</Warning>
```

Comment:
maybe it would be useful to show an example of the callback URL like so
`https://192-168-1-108.auth.local.nhost.run/v1/signin/provider/google/callback`

Reply:
I think that will just clutter the docs and will probably lead to some users copy and pasting (we already had an example yesterday). A reminder will probably make them have to think about it rather than follow blindly and paste the wrong thing.

Reply:
https://github.com/nhost/nhost/issues/2821

Reply:
sounds good 👌